src/frontend/src/catalog/schema_catalog.rs

```diff
@@ -37,7 +37,8 @@ pub struct SchemaCatalog {
     table_name_by_id: HashMap<TableId, String>,
     source_by_name: HashMap<String, SourceCatalog>,
     source_name_by_id: HashMap<SourceId, String>,
-    sink_bimap: BiMap<SinkId, String>,
+    sink_by_name: HashMap<String, SinkCatalog>,
```

Comment:
Bimap was to eliminate the duplication of the two maps. I personally prefer the former.

Reply:
Bimap is not suitable here, we need more information rather than only id and name here for sink.